### PR TITLE
Replacing stubs with opensource

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

**Comment by dNechita** [comment]
The source files for the stubs will no longer be needed. So delete them, please.

File: scripts/windows/setup_project.bat [hunk 1]
```diff
@@ -234,15 +222,11 @@ CALL :install_websockets %config_type% %generator%
...
-   cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DUSE_DEPTH_COMPUTE_STUBS=ON -DCMAKE_BUILD_TYPE=%config_type%
-   ) else (
-		if %use_depth_compute_opensource%==1 (
-			cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DUSE_DEPTH_COMPUTE_OPENSOURCE=ON -DCMAKE_BUILD_TYPE=%config_type%
-		) else (
-			cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DCMAKE_BUILD_TYPE=%config_type%
-		) 
-)
+if %use_depth_compute_opensource%==1 (
+    cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DUSE_DEPTH_COMPUTE_OPENSOURCE=ON -DCMAKE_BUILD_TYPE=%config_type%
```
Comment (dNechita) [inline scripts/windows/setup_project.bat:226]: On host (Windows, Linux, etc.) the  depth compute is no longer needed. Please remove flag DUSE_DEPTH_COMPUTE_OPENSOURCE=ON.

Comment (dNechita) [reply scripts/windows/setup_project.bat:226]: Also remove the option from setup_project script.

Comment (liealex) [reply scripts/windows/setup_project.bat]: Solved

**Pull request merged**